src/core/packet_space.h

```diff
@@ -104,6 +104,11 @@ typedef struct QUIC_PACKET_SPACE {
     //
     BOOLEAN AwaitingKeyPhaseConfirmation: 1;
 
+    //
+    // ECN marks of received packets.
+    //
+    QUIC_ACK_ECN_EX RecvECN;
```

Comment:
This is already being tracked in the ACK tracker. This should be unnecessary.

Reply:
Good catch. Fixed.